receptai.api/Repositories/SubfoodditRepository.cs

```diff
@@ -0,0 +1,83 @@
+﻿using Microsoft.EntityFrameworkCore;
+using receptai.api.Dtos.Recipe;
+using receptai.api.Dtos.Subfooddit;
+using receptai.api.Helpers;
+using receptai.api.Interfaces;
+using receptai.data;
+
+namespace receptai.api.Repositories;
+
+public class SubfoodditRepository : ISubfoodditRepository
+{
+    private readonly RecipePlatformDbContext _context;
+
+    public SubfoodditRepository(RecipePlatformDbContext context)
+    {
+        _context = context;
+    }
+
+    public async Task<Subfooddit> CreateAsync(Subfooddit subfoodditModel)
+    {
+        await _context.Subfooddits.AddAsync(subfoodditModel);
+        await _context.SaveChangesAsync();
+
+        return subfoodditModel;
+    }
+
+    public async Task<Subfooddit?> DeleteAsync(int id)
+    {
+        var subfoodditModel = await _context.Subfooddits
+            .FirstOrDefaultAsync(r => r.SubfoodditId == id);
```

Comment:
Should use "s" instead of "r". "r" stands for "recipe", does not make much sense here